content/services/csv/powerdev_gpu_projects.csv

```diff
@@ -1,6 +1,7 @@
-"CASHX Pipeline","Sequence alignment tools CASHX and pipeline were ported to work on the IBM POWER8 to take full advantage of higher thread count machines. CASHX is one of the few non-BWA based high throughput genome alignment tools."
 "CADA","CADA - CUDA Accelerated DNA Assembly is being developed by Ryan Kitchen to change the way genome assembly uses hardware resources."
+"CASHX Pipeline","Sequence alignment tools CASHX and pipeline were ported to work on the IBM POWER8 to take full advantage of higher thread count machines. CASHX is one of the few non-BWA based high throughput genome alignment tools."
 "CASSA","CASSA - CUDA Accelerated Scalable Sequence Aligner. The CGRB is working with Ryan Kitchen to develop this new alignment tool."
+"Cryo-EM","Frank Liu and Suvadip Paul are using GPU Acceleration on Power8 and Power9 architecture to build, debug, and test software used to examine and model highly detailed microscopic structures such as proteins gathered using Cryo-EM technology."
```

Comment:
Please use "POWER8" or "POWER9" (i.e. all caps).

Reply:
I update the site for all misuses of Power.